src/service-api.ts

```diff
@@ -92,13 +93,21 @@ const plugin: FastifyPluginAsync = async (fastify) => {
 
   // get available tags
   fastify.get(
-    '/tags', { schema: getTags },
+    '/tags/list', { schema: getTags },
     async ({ member, log }) => {
       const task = taskManager.createGetAvailableTagsTask(member);
       return runner.runSingle(task, log);
     }
   );
 
+  fastify.get<{ Querystring: IdsParams }>(
+    '/tags', { schema: getMany }, 
+    async ({ member, query: { id: ids }, log }) => {
+      const tasks = ids.map(id => taskManager.createGetOfItemTask(member, id));
```

Comment:
As discussed, let's try to have a an array of objects, and without duplicates. 
And is there room for optimisation? (eg: one cal instead of n calls). If it will take to much we can leave a todo for the future.